src/bin/repl/mod.rs

```diff
@@ -97,6 +97,56 @@ impl<'s> Repl<'s> {
     }
 
     fn execute_code(&mut self, code: &str) {
-        let program = check_semantics(code, self.opt.clone());
+        let module = initialize_jit_module();
+        let expr = match analyze(code, Parser::expr, PureAnalyzer::expr) {
+            Ok(mut expr) => {
+                expr.ctype = types::Type::Int(true);
```

Comment:
This doesn't look right. What's this meant to do?

Reply:
Yeah, this crashes on strings:
```
>> "a"
The application panicked (crashed).
Message:  verification error: - inst1 (return v0): arg 0 (v0) has type i64, must match function signature of i32

note: while compiling function u0:0() -> i32 system_v {
    gv0 = symbol colocated u1:0

block0:
    v0 = global_value.i64 gv0
    return v0
}
```

Reply:
Instead you could use a function name other than `main` and set the return type to the type of the expression. Then you decide how to display the result depending on the type. That would also fix the current bug where `'a'` is displayed as `99`.

Reply:
That's a good idea

Reply:
I had this code there, because otherwise I got an error from cranelift that it expected an i32 but got i64.